src/Dropdown/README.md

```diff
@@ -14,3 +14,9 @@ Note: When passing in a component to `menuItems` that uses a prop other than `re
 
 ### `title` (string; required)
 `title` specifies the text that is displayed within the original dropdown button.
+
+### `iconElement` (element; optional)
+`iconElement` specifies the icon to be displayed alongside the dropdown. It can be one of
```

Comment:
`iconElement` could technically be any element, not just an `<img />` or `<Icon />`. For example, you might want to wrap an `<img />` in a router `Link` component.